src/test/java/org/gridsuite/shortcircuit/server/service/AnalysisParametersTest.java

```diff
@@ -0,0 +1,6 @@
+package org.gridsuite.shortcircuit.server.service;
+
+import org.assertj.core.api.WithAssertions;
+
+class AnalysisParametersTest implements WithAssertions {
```

Comment:
Hum ??

Reply:
oups, that was the class where I initially tested parameters related functions.